.travis-setup.sh

```diff
@@ -17,42 +17,49 @@
 #  along with this program; if not, write to the Free Software
 #  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 
-glib_version=2.49.4
+set -e -x
+
+glib_version=2.46.2
```

Comment:
I'm not sure we want to do this - the minimum version we support is 2.46, but we don't have to build with the absolute minimum do we? Or are we doing so to guarantee that the version in `configure.ac` is still correct? If the latter, we need a comment.

Also, 2.49.4 seems to be the dev version, and 2.46.2 is an old version - how about 2.48.1 (latest stable)? :-)